fire_atlas_editor/lib/widgets/container.dart

```diff
@@ -1,12 +1,12 @@
 import 'package:flutter/material.dart';
 
 class FContainer extends StatelessWidget {
-  final double width;
-  final double height;
-  final Widget child;
-  final Color color;
-  final EdgeInsets margin;
-  final EdgeInsets padding;
+  final double? width;
+  final double? height;
+  final Widget? child;
+  final Color? color;
+  final EdgeInsets? margin;
```

Comment:
This has a default, so shouldn't need to be nullable right? Maybe `padding` could also have a default?